src/commands/create.c

```diff
@@ -22,6 +22,10 @@
 
 extern struct start_data start_data;
 
+/* ignore -pedantic to cast handle_option_console, a function pointer, to a
+ * void* */
+#pragma GCC diagnostic push
```

Comment:
I guess the answers to those questions will have to come with time. Worse than can happen is being ignored though, maybe with a warning.

The good news is that it's now also working with clang.